serverless.yml

```
@@ -9,7 +9,10 @@ provider:
   versionFunctions: false
   region: ${opt:region, "us-east-1"}
   stage: ${opt:stage, "local"}
-  memorySize: 512
+  memorySize: 2048
```

```suggestion
  memorySize: 2048 # 2gb
```
On the one hand, it feels silly to document this, on the other hand, the unit isn't listed, and it's easy to think the unit might be kb, byte, or even bits.
Same with next line --- I just was not sure how to annotate that.